lib/datadog/notifications/plugins.rb

```diff
@@ -5,6 +5,6 @@ module Plugins
   end
 end
 
-%w[base grape active_record active_job action_controller].each do |name|
+%w[base grape active_record active_job action_controller grpc].each do |name|
```

Comment:
instead, can we do this:

```
 module Datadog
   class Notifications
     module Plugins
       autoload :Base, 'datadog/notifications/plugins/base'
       autoload :ActiveRecord, 'datadog/notifications/plugins/active_record'
       ...
     end
  end
end
```

Reply:
Btw, why `autoload :Name, 'path/to/file'`? Looks like everything's put in right place so should be resolved just fine, no?